src/main/java/com/opencart/repository/LoginModelRepository.java

```diff
@@ -0,0 +1,48 @@
+package com.opencart.repository;
+
+import com.opencart.datamodel.LoginModel;
+
+public class LoginModelRepository {
+
+    private LoginModelRepository() {
+    }
+
+    public static LoginModel getValidLoginModel() {
+        LoginModel loginModel = new LoginModel();
```

Comment:
You do not need to create a new object here.
Just do the following:
.email("someEmail")
.password("somePassword")

Reply:
Done